### ISSUE TITLE
Proper legend position

### ISSUE THREAD

**Issue by alifbe** [equinor/xtgeoviz#32]
Legend position get weird when plotting 3D grid property in xtgeoviz.

It will be great if for example the color bar can be positioned above well map and both zone log and perforation log remained on the right end.

![MLW_OP5_Y2_xsection](https://github.com/equinor/xtgeoviz/assets/11570927/b16bbab9-e067-450f-b66e-0d8087d3b45c)

![MLW_OP5_Y2_SOIL_20200701_xsection](https://github.com/equinor/xtgeoviz/assets/11570927/8bd08a77-9e24-471b-82d6-14bb1f65605b)

**Comment by jcrivenaes** [comment]
Also related here is that Zones and Zonelog plots in top of each other. Prior to version 0.1.0, the Zonelog was not plotted

![image](https://github.com/equinor/xtgeoviz/assets/16700455/d5d33f91-d551-44cb-ade7-2789573dcd79)

**Comment by alifbe** [comment]
Yes, I think I saw something similar. If I remember correctly, I turn off plot legend option to surfaces in Drogon example.
